Base/usr/share/man/man7/Tips-and-Tricks.md

```diff
@@ -0,0 +1,63 @@
+![Welcome Icon](/res/icons/32x32/app-welcome.png)
+
+## Name
+Tips and Tricks
+
+## Description
+This is a list of useful tips and tricks to help you make the most out of SerenityOS.
+
+## General
+* When on the Desktop or in File Manager, start typing the name of an item to select it.
+* Bold text in context menus hints at the default behavior of a double-click.
+* Hold `Ctrl` to accelerate mouse wheel interaction with sliders and spin boxes.
+* Hold `Ctrl` while activating a menu item to prevent that menu from closing.
+* Many applications can open a compatible file if you drag-and-drop it into their window.
+* Change default file and protocol associations in `~/.config/LaunchServer.ini`.
+
+### Window Management
+* Double-click a window's title bar to maximize it.
+* Click on a window's icon to open the window's context menu.
+* Double-click on the edge of an application's window to maximize it in that direction.
+* Middle-click on a window's maximize button to extend the window vertically (this can be undone in the same way).
+* Drag resizable windows to any side or corner of the screen to automatically resize them to fill half or one-quarter of the screen.
+* Right-click on the Workspace Picker applet and select 'Workspace Settings' to easily customize the number and layout of Workspaces (virtual desktops).
+
+### Fun
+* It can help to get a second pair of `$ Eyes` on a problem… or fifty: `$ Eyes -n 100`.
+
+## Applications
+
+### [Assistant](help://man/1/Applications/Assistant)
+* Assistant can help you to quickly find files and launch applications. Open it with `Super+Space`.
+* Enter a URL to open it in the web browser.
+* Perform quick calculations by prefixing your expression with the equal sign (`=`), e.g. `=2*5`.
+    * Pressing Return will copy the result to the clipboard.
```

Comment:
i feel you can merge it with a previous (use full stop or semicolon to separate?).